### PR TITLE
View Users

### PR DESCRIPTION
**Pull request by Rawan96**
Adding the back-end route (/event/:eventcode/users) to get the users of a special event. By creating a query which give us the data of the users and I give it a condition that allow me to get the users of a particular event (event_code).

### CODE REVIEW

**Comment by AhmedSafi97** [comment]
**where is PR description ?**

File: server/database/queries/getUsersOfEvent.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const connection = require('../config/connection');
+
+const getUsersOfEvent = (eventCode) => connection.query('SELECT users.first_name, users.last_name, users.mobile, users.email,users.location FROM users INNER JOIN user_event ON users.id = user_event.user_id INNER JOIN events ON user_event.event_id = events.id WHERE events.event_code = 1$ ', [eventCode]);
+
+module.exports = getUsersOfEvent;
```
Comment (AhmedSafi97) [inline server/database/queries/getUsersOfEvent.js]: nice query ! good job :grinning: 
**BUT** what about  attendance from the user_event table ? I guess we need to know  which users did really attend the event and who  just booked it without attendance

**Comment by Rawan96** [comment]
> **where is PR description ?**

Sorry this is by mistake, I added the description of the PR

File: server/database/queries/getUsersOfEvent.js (hunk 1 above)
Comment (Rawan96) [reply server/database/queries/getUsersOfEvent.js]: Thank's Ahmed,
Yeah actually this is a good point, I added the attendance to the query.

Comment (AhmedSafi97) [reply server/database/queries/getUsersOfEvent.js]: Your welcome, keep up the hard work :+1:

**Pull request merged**